src/Discussion/Search/Driver/MySqlDiscussionDriver.php

```diff
@@ -9,25 +9,34 @@
  * file that was distributed with this source code.
  */
 
-namespace Flarum\Discussion\Search\Fulltext;
+namespace Flarum\Discussion\Search\Driver;
 
 use Flarum\Post\Post;
+use Flarum\Discussion\Discussion;
 
-class MySqlFulltextDriver implements DriverInterface
+class MySqlDiscussionDriver implements DriverInterface
 {
     /**
      * {@inheritdoc}
      */
     public function match($string)
     {
-        $discussionIds = Post::where('type', 'comment')
+        $contentDiscussionIds = Post::where('type', 'comment')
             ->whereRaw('MATCH (`content`) AGAINST (? IN BOOLEAN MODE)', [$string])
             ->orderByRaw('MATCH (`content`) AGAINST (?) DESC', [$string])
             ->lists('discussion_id', 'id');
+        $titleDiscussionIds = Discussion::where('is_approved', 1)
+            ->whereRaw('MATCH (`title`) AGAINST (? IN BOOLEAN MODE)', [$string])
+            ->orderByRaw('MATCH (`title`) AGAINST (?) DESC', [$string])
+            ->lists('id');
```

Comment:
I'm guessing the `orWhereRaw` should MATCH content, instead of title, right?

Ugh, now I gotta create a new Flarum installation because I don't feel like changing all files to next-back. 😁